packages/salesforcedx-vscode-lwc-next/package.json

```diff
@@ -25,9 +25,10 @@
   "dependencies": {
     "@salesforce/salesforcedx-utils-vscode": "43.14.0",
     "ajv": "^6.1.1",
-    "eslint": "4.16.0",
-    "eslint-plugin-lwc": "0.4.0",
-    "lwc-language-server": "1.6.6",
+    "eslint-plugin-lwc":
```

Comment:
@lcampos - npm also has a feature to specify a tarball URL. See https://docs.npmjs.com/files/package.json#urls-as-dependencies

So I used that instead of bundling it directly. I got this url using, e.g., `npm view --registry=https://npm.lwcjs.org/ lwc-language-server@1.6.1`